Tests/test_CodonTable.py

```diff
@@ -667,6 +668,29 @@ def test_table31(self):
         self.assertEqual(amb_dna_table.forward_table["TAR"], "E")
         self.assertEqual(nuc_table.forward_table["UGA"], "W")
 
+    def test_table32(self):
+        """Check table 32: Balanophoraceae Plastid.
+
+        NOTE: This code has one unambiguous stop codon! TAA codes for either
+        stop or W, dependent on the context.
+        Another note: I guess this is wrong. There is a recent publication
+        that shows that TAA is not a stop codon anymore but only coding for
+        W. So table 32 may change in the near future.git 
```

Comment:
Is the "git" here from typing in the wrong window, or a tongue in cheek comment?

Reply:
:-(, yes, together with a trailing whitespace....

Reply:
I introduced this while removing a whitespace issue, so the new whitespace warning during the commit passed my attention....